docs/advanced_topics/accessibility_considerations.rst

```diff
@@ -0,0 +1,163 @@
+Accessibility considerations
+============================
+
+Accessibility for CMS-driven websites is a matter of :ref:`modeling content appropriately <content_modeling>`, :ref:`creating accessible templates <accessibility_in_templates>`, and :ref:`authoring content <authoring_accessible_content>` with readability and accessibility guidelines in mind.
+Wagtail generally puts developers in control of content modeling and front-end markup, but there are a few areas to be aware of nonetheless, and ways to help authors be aware of readability best practices.
```

Comment:
Maybe reword/extend this introduction and make clear this page is about what Wagtail can do to make content accessible. Other topics (contrast, font-size, valid markup) are important but not discussed.